pyproject.toml

```diff
@@ -865,6 +865,10 @@ openlineage = [ # source: airflow/providers/openlineage/provider.yaml
 opensearch = [ # source: airflow/providers/opensearch/provider.yaml
   "opensearch-py>=2.2.0",
 ]
+opentelemetry = [ # source: airflow/providers/opentelemetry/provider.yaml
```

Comment:
We already have `[otel]` in the core extras, and having two of them will be confusing (see beginning of the optionsal dependencies)

I guess it makes sense to make `otel` a deprecated alias to `opentelemetry` and add the prometheus dependency here?

Reply:
Right, it did have collision when I tried to use `otel` for the providers. So, I instead opted to use `opentelemetry` for providers, and `otel` for airflow core opentelemetry. I'm not sure I know how to combine the name of both to `opentelemetry` - might need someone else's help on getting the names squared away.